content/en/docs/installation/kubectl.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Kubectl apply"
+linkTitle: "Kubectl apply"
+weight: 20
+type: "docs"
+---
+
+## Installing with regular manifests
+
+### Prerequisites
+
+- `kubectl` version `>= v1.19.0-rc.1` (otherwise, you will have issues updating the CRDs. see [v0.16 upgrade notes](../upgrading/upgrading-0.15-0.16/#issue-with-older-versions-of-kubectl))
+- A Kubernetes or OpenShift cluster running a [supported version](../supported-releases/)
+- cert-manager not already installed on the cluster
+- [Prerequisites specific to Cloud provider](../compatibility/)
+
+### Steps
+
+All resources (the [`CustomResourceDefinitions`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) and the cert-manager, cainjector and webhook components)
+are included in a single YAML manifest file:
+
+Install all cert-manager components:
+
+```bash
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.yaml
+```
+
+> **Note**: When running on GKE (Google Kubernetes Engine), you may encounter a
+> 'permission denied' error when creating some of these resources. This is a
+> nuance of the way GKE handles RBAC and IAM permissions, and as such you should
+> 'elevate' your own privileges to that of a 'cluster-admin' **before** running
+> the above command. If you have already run the above command, you should run
+> them again after elevating your permissions:
+```bash
+$ kubectl create clusterrolebinding cluster-admin-binding \
+    --clusterrole=cluster-admin \
+    --user=$(gcloud config get-value core/account)
+```
```

Comment:
suggestion: it doesn't feel quite right having a cloud-provider-specific suggestion here in this block. Could we maybe add a new subheading under the line `Once you have deployed cert-manager, you can verify the installation [here](../verify/).` below?

```suggestion
### Permissions Errors on Google Kubernetes Engine

When running on , you might encounter a 'permission denied' error when creating some
of the required resources. This is a nuance of the way GKE handles RBAC and IAM permissions,
and as such you might need to elevate your own privileges to that of a "cluster-admin" **before**
running `kubectl apply`.

If you have already run `kubectl apply`, you should run it again after elevating your permissions:

```bash
$ kubectl create clusterrolebinding cluster-admin-binding \
    --clusterrole=cluster-admin \
    --user=$(gcloud config get-value core/account)
\``` # 🔥 should just be three backticks but it's hard to type three backticks inside a suggestion block 😂
```